packages/wonder-blocks-form/src/components/labeled-text-field.js

```diff
@@ -53,25 +97,42 @@ export default class LabeledTextField extends React.Component<Props, State> {
         });
     };
 
+    handleOnChange: (newValue: string) => mixed = (newValue) => {
+        this.setState({
+            value: newValue,
+        });
+    };
+
     render(): React.Node {
-        const {disabled} = this.props;
+        const {id, type, label, description, disabled} = this.props;
+
         return (
-            <FieldHeading
-                field={
-                    <TextField
-                        id="tf-1"
-                        value="Value"
-                        placeholder="Placeholder"
-                        disabled={disabled}
-                        onChange={() => {}}
-                        onFocus={this.handleOnFocus}
-                        onBlur={this.handleOnBlur}
+            <IDProvider id={id} scope="field">
```

Comment:
If I remember correctly, the scope will be prefixed as part of the generated unique id, right?

Reply:
The scope is added in the middle of the unique id:

<img width="955" alt="idprovider-example" src="https://user-images.githubusercontent.com/60367213/122258752-05587980-ce97-11eb-9bf3-eb17f7e062b9.png">

Reply:
I also have now changed the scope to be `"labeled-text-field"` to be more descriptive.